src/utils/formatting/index.ts

```diff
@@ -0,0 +1,67 @@
+import strip from 'strip-ansi';
+import {PADDING_SIZE, PADDING} from '../../constants';
```

Comment:
We don't need a folder for just an simple index.ts, `src/utils/formatting` will do the work just fine
```suggestion
import {PADDING_SIZE, PADDING} from '../constants';
```